### ISSUE TITLE
bug in version compare

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#202]
On `devon ide update` I got this:
```
You are using version 3.0.0-beta9 of devon-ide-scripts
The new version to install is 3.0.0-beta14
You are using a newer version than the latest release version.
Hence there is nothing to update.
```

So it seems that `doVersionCompare` is buggy:
https://github.com/devonfw/devon-ide/blob/c1eea27ef10e9cc21dc5a870f31d74f6e8efac88/scripts/src/main/resources/scripts/functions#L883

**Comment by hohwille** [comment]
Fixed with PR #201

**Comment by hohwille** [comment]
```
Trying to determine the latest available version from https://repo.maven.apache.org/maven2/com/devonfw/tools/ide/devon-ide-scripts/maven-metadata.xml
LATEST = 3.0.0-beta17
You are using version 3.0.0-beta11-SNAPSHOT of devon-ide-scripts
The new version to install is 3.0.0-beta17
You are using a newer version than the latest release version.
Hence there is nothing to update.
Seems as you are an active devonfw developer. Thanks for contributing!
```
I copied the functions from the latest `devon-ide` to that old installation (3.0.0-beta11-SNAPSHOT). So I have to verify if it is still buggy or still some old code from beta11 interferes.

**Comment by hohwille** [comment]
Ouch:
```
3.0.0-beta17 > 3.0.0-beta11-SNAPSHOT ? FAILED! Was: 3.0.0-beta17 < 3.0.0-beta11-SNAPSHOT
```
Reopening.

**Comment by hohwille** [comment]
Fixed with PR #218
